src/IceRpc.Protobuf.Generators/Internal/Emitter.cs

```diff
@@ -82,7 +82,19 @@ request.Operation switch
             generatedClasses.Add(container);
         }
 
-        string generated = "using IceRpc.Protobuf;\n\n";
+        string generated = @$"
+// <auto-generated/>
```

Comment:
Should we do the same for IceRpc.Slice.Generators?

Reply:
Yes, I created an issue for this.